docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: docs
+title: Command Line Interface
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+TestCafe command line interface is pretty simple.
+It consists of a single command that runs tests with a number of options.
+
+## Synopsis
+
+```sh
+testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
+```
+
+## Browser List
+
+### Local Browsers
+
+Specify the list of browsers (separated by commas) where tests will be ran.
+You can specify browser paths or aliases. 
```

Comment:
The term `alias` first appears here without any explanation. I feel like we need separate section for the aliases, which we can cross-link both from here, `list-browsers` and programmatical API section. \cc @VasilyStrelyaev